test/norm_num.lean

```diff
@@ -521,18 +521,6 @@ section
   example : - (-4 / 3) = 1 / (3 / (4 : α)) := by norm_num1
 end
 
-section Transparency
-
-example : Add.add 10 2 = 12 := by norm_num1
-example : Nat.sub 10 1 = 9 := by norm_num1
-example : Nat.mod 10 5 = 0 := by norm_num1
-example : Sub.sub 10 1 = 9 := by norm_num1
-example : Sub.sub 10 (-2) = 12 := by norm_num1
-example : Mul.mul 10 1 = 10 := by norm_num1
-example : (Div.div 10 1 : ℚ) = 10 := by norm_num1
-
-end Transparency
-
```

Comment:
These were added without explanation in https://github.com/leanprover-community/mathlib4/commit/60790d4fd15e2d67606041b9f7058872911a8ac8#r138404530

Reply:
I have a feeling they came up during testing and they thought it'd just be easy to support them.

Reply:
It's possible that refactors elsewhere made this behavior no longer necessary